marketpulse/base/mozillians.py

```diff
@@ -0,0 +1,85 @@
+import requests
+
+
+class BadStatusCode(Exception):
+    pass
+
+
+class ResourceDoesNotExist(Exception):
+    pass
+
+
+class MultipleResourcesReturned(Exception):
+    pass
+
+
+class MozilliansClient():
+    def __init__(self, api_base_url, api_key):
+        self.base_url = api_base_url
+        self.key = api_key
+        self.headers = {'X-API-KEY': self.key}
+
+    def _get(self, url, params=None):
+        """GET url, checks status code and return ApiResponse."""
+        response = requests.get(url, headers=self.headers, params=params)
+
+        if not response.status_code == 200:
+            raise BadStatusCode('{0} on: {1}'.format(response.status_code, url))
+
+        return response.json()
+
+    def _get_resource_url(self, resource):
+        """Build Mozillians.org API resource url."""
+        url = '{0}/{1}'.format(self.base_url, resource)
+        return url
+
+    def get_users(self):
+        """GET Mozillians.org users."""
+        url = self._get_resource_url('users')
+        return self._get(url)
+
+    def get_groups(self):
+        """GET Mozillians.org groups"""
+        url = self._get_resource_url('groups')
+        return self._get(url)
+
+    def get_skills(self):
+        """GET Mozillians.org skills."""
+        url = self._get_resource_url('skills')
+        return self._get(url)
+
+    def get_user(self, user_id):
+        """GET Mozillians.org user by id."""
+        url = self._get_resource_url('users/{0}/'.format(user_id))
+        return self._get(url)
+
+    def get_group(self, group_id):
+        """GET Mozillians.org group by id."""
+        url = self._get_resource_url('groups/{0}/'.format(group_id))
+        return self._get(url)
+
+    def get_skill(self, skill_id):
+        """GET Mozillians.org skill by id."""
+        url = self._get_resource_url('skills/{0}/'.format(skill_id))
+        return self._get(url)
```

Comment:
I am not sure that `get_user`, `get_group`, `get_skill` are really helpful right now - maybe they should just `_get` the absolute url and not the id.